instance/dig.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2021 The Dank Grinder authors.
+//
+// This source code has been released under the GNU Affero General Public
+// License v3.0. A copy of this license is available at
+// https://www.gnu.org/licenses/agpl-3.0.en.html
+//
+// Dig Functionality added by https://github.com/V4NSH4J
+
+package instance
+
+import (
+	"math/rand"
+	"regexp"
+
+	"github.com/dankgrinder/dankgrinder/discord"
+	"github.com/dankgrinder/dankgrinder/instance/scheduler"
+)
+
+func isSameLength(s1 string, s2 string) bool { // Function checks length of strings
+
+	if len(s1) == len(s2) {
+		return true
+	} else {
+		return false
+	}
+}
+
+func hasSameLetters(s1 string, s2 string) bool { // Function checks the letters of the strings (Scrambled and Unscrambled Words)
+	var counter int
+	for _, letter1 := range s1 {
+		for _, letter2 := range s2 {
+			if letter1 == letter2 {
+				counter++
+				break
+			}
+		}
+	}
+	if counter == len(s1) {
+		return true
+	} else {
+		return false
+	}
+}
+
+func find(tt string, stuff []string) (string, string) { //Some amazing stolen/borrowed code which finds the missing word and phrase from a list of pre-determined phrases
```

Comment:
Same about the comment. Also there is no need to document that you stole/borrowed code. 

Furthermore dislike the argument name `stuff`.

Reply:
haha, I was thinking the variable "list" first, then I thought it was too generic and might cause some confusion, so I settled with stuff. Anyways changed it to listOfPhrases obeying CamelCode :)